core/report-l10n.rb

```diff
@@ -170,7 +170,7 @@ module ISO639
 dar|||Dargwa|dargwa
 day|||Land Dayak languages|dayak, langues
 del|||Delaware|delaware
-den|||Slave (Athapascan)|esclave (athapascan)
+den|||Slavic (Athapascan)|slave (athapascan)
```

Comment:
Revert, see https://fr.wikipedia.org/wiki/Esclave_(langue)

Reply:
yes my bad it seems to be a North America language...I bet there is an history behind, but you are right. Sorry :D

Reply:
```suggestion
den|||Slave (Athapascan)|esclave (athapascan)
```
```